packages/docker.rb

```diff
@@ -0,0 +1,177 @@
+  homepage 'https://www.docker.com/'
+  version '24.0.1'
+  license 'Apache'
+  compatibility 'x86_64 aarch64 armv7l'
+  source_url 'https://github.com/docker/cli.git'
+  git_hashtag "v#{version}"
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/docker/24.0.1_armv7l/docker-24.0.1-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/docker/24.0.1_armv7l/docker-24.0.1-chromeos-armv7l.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/docker/24.0.1_x86_64/docker-24.0.1-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: '923b0eb95c947dde422e196abd0f1ddbc0958792db2807e2c55c3048b700702a',
+     armv7l: '923b0eb95c947dde422e196abd0f1ddbc0958792db2807e2c55c3048b700702a',
+     x86_64: '21f9db6ec0086ab6452be9cd50a6bc980eb00caafb439a3387802c5363085ec1'
+  })
+
+  depends_on 'bridge_utils'
+  depends_on 'btrfsprogs' => :build
+  depends_on 'containerd'
+  depends_on 'elogind' => :build
+  depends_on 'eudev'
+  depends_on 'glibc' # R
+  depends_on 'go' => :build
+  depends_on 'go_md2man' => :build
+  depends_on 'iproute2'
+  depends_on 'lvm2' # R
+  depends_on 'sqlite'
+
+  no_env_options
+  no_fhs
+
+  def self.build
+    @cli_version = git_hashtag
+
+    binary_url({
+      aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/docker/24.0.1_armv7l/docker-24.0.1-chromeos-armv7l.tar.zst',
+       armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/docker/24.0.1_armv7l/docker-24.0.1-chromeos-armv7l.tar.zst',
+       x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/docker/24.0.1_x86_64/docker-24.0.1-chromeos-x86_64.tar.zst'
+    })
+    binary_sha256({
+      aarch64: '923b0eb95c947dde422e196abd0f1ddbc0958792db2807e2c55c3048b700702a',
+       armv7l: '923b0eb95c947dde422e196abd0f1ddbc0958792db2807e2c55c3048b700702a',
+       x86_64: '21f9db6ec0086ab6452be9cd50a6bc980eb00caafb439a3387802c5363085ec1'
+    })
+
+    @moby_version = git_hashtag
+
+    binary_url({
+      aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/docker/24.0.1_armv7l/docker-24.0.1-chromeos-armv7l.tar.zst',
+       armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/docker/24.0.1_armv7l/docker-24.0.1-chromeos-armv7l.tar.zst',
+       x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/docker/24.0.1_x86_64/docker-24.0.1-chromeos-x86_64.tar.zst'
+    })
+    binary_sha256({
+      aarch64: '923b0eb95c947dde422e196abd0f1ddbc0958792db2807e2c55c3048b700702a',
+       armv7l: '923b0eb95c947dde422e196abd0f1ddbc0958792db2807e2c55c3048b700702a',
+       x86_64: '21f9db6ec0086ab6452be9cd50a6bc980eb00caafb439a3387802c5363085ec1'
+    })
+
+    @libnetwork_version = '3f0048413d95802b9c6c836eba06bfc54f9dbd03'
+    @tini_version = '0b44d3665869e46ccbac7414241b8256d6234dc4'
+    @buildx_version = 'v0.10.5'
```

Comment:
Should these be separate dependency packages?

Reply:
Nothing else uses them... and they all get built together...

Reply:
We'll just have to remember to increment the versions on updates, if necessary.